src/config/index.js

```diff
@@ -0,0 +1,5 @@
+require('dotenv').config();
+
+module.exports.server = {
+  PORT: process.env.PORT,
+};
```

Comment:
You have a config! Great work!
But there is also `ORIGIN` variable. It should be in the config too.

Also, when you have a complex object to export try to make a variable for it, then export that variable.
Examples:
```
module.exports = {
  functionOne,
  functionTwo,
  functionThree,
};
```
```
const config = {
  server: {
    port: process.env.PORT || '3000',
    hots: process.env.HOST || 'localhost',
  },
};

module.exports = config;
```